pkg/controller/gkenetworkparamset/gnpcontroller_validations.go

```diff
@@ -233,3 +239,77 @@ func crossValidateNetworkAndGnp(network *networkv1.Network, params *networkv1.GK
 		IsValid: true,
 	}
 }
+
+// hasNewPodRange returns true if the node has new Pod range that's not in the "default" params
+func (c *Controller) hasNewPodRange(node *v1.Node) bool {
```

Comment:
this name is confusing, `hasNonDefaultPodRange` or something like that sounds more accurate from the description